React-Calculator/src/components/WolfButtonGrid.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+
+export function WolfButtonGrid({ handleOnClick }) {
+    return (
+        <div className="w-[425px] h-[373px] px-2.5 py-[30px] justify-start items-start gap-[25px] inline-flex items-center">
+            <div className="justify-start items-start gap-[25px] inline-flex flex-wrap items-center">
+                <div className="justify-start items-start gap-[25px] inline-flex items-center">
+                    <Button valueType="operand" value="Off" handleClick={handleOnClick} />
+                    <Button valueType="operand" value="C" handleClick={handleOnClick} />
+                    <Button valueType="operand" value="del" handleClick={handleOnClick} />
+                </div>
+                <div className="justify-start items-start gap-[25px] inline-flex flex-wrap items-center">
+                    {[7, 8, 9, 4, 5, 6, 1, 2, 3, '.', 0, '='].map((value) => (
+                        <Button valueType="number" value={value} handleClick={handleOnClick} key={`button${value}`} />
+                    ))}
+                </div>
+            </div>
```

Comment:
this could probably be a very small component. It should be consistent (if not, exactly the same) as the one in the regular calculator, right?